libs/core/src/byte_array/encoders.cpp

```diff
@@ -29,11 +29,6 @@ namespace byte_array {
 
 ConstByteArray ToBase64(ConstByteArray const &str)
 {
-  if (str.empty())
```

Comment:
If this check is removed so the return value is always `ret`, it would be nice to initialize that variable, check its size and return if empty first thing, before executing casts and a myriad of other operations.

Reply:
This peiece of the code will go back, it has been removed by mistake.